rewrite.yml

```diff
@@ -1,13 +1,16 @@
 ---
 type: specs.openrewrite.org/v1beta/recipe
-name: org.checkstyle.AllAutoFixes
-displayName: Auto Fixes
-description: List of auto fixes from different providers.
+name: org.checkstyle.openrewrite.SanityCheck
+displayName: Apply best practices
+description: Comprehensive code quality recipe combining modernization and best practices.
 recipeList:
   - org.checkstyle.autofix.CheckstyleAutoFix:
       violationReportPath: "target/cs_errors.xml"
       configurationPath: "config/checkstyle-checks.xml"
       propertiesPath: "config/openrewrite-recipes-checkstyle.properties"
+  - org.openrewrite.codemods.ESLint
+  - org.openrewrite.codemods.cleanup.javascript.EmptyBraceSpaces
```

Comment:
will help us in near future just like:

- https://github.com/checkstyle/checkstyle/issues/18198